### PR TITLE
Re render

### PR DESCRIPTION
**Pull request by ocefpaf**
Ping @snowman2.

Alternative to #135

### CODE REVIEW

**Comment by ocefpaf** [comment]
`test_mp_main_env` is segfaulting on Linux and Windows has 23(!) tests failing. Some are odd failures like:

```
2019-08-11T17:06:21.0729521Z ________________________ test_transform_point_src_file ________________________
2019-08-11T17:06:21.0729594Z 
2019-08-11T17:06:21.0729682Z     def test_transform_point_src_file():
2019-08-11T17:06:21.0729782Z         runner = CliRunner()
2019-08-11T17:06:21.0729870Z         result = runner.invoke(main_group, [
2019-08-11T17:06:21.0729969Z             'transform',
2019-08-11T17:06:21.0730214Z             '--src-crs',
2019-08-11T17:06:21.0730288Z             'tests/data/RGB.byte.tif',
2019-08-11T17:06:21.0730381Z             '--precision', '2'
2019-08-11T17:06:21.0730572Z         ], "[192457.13, 2546667.68]")
2019-08-11T17:06:21.0730667Z         assert result.exit_code == 0
2019-08-11T17:06:21.0730745Z >       assert result.output.strip() == '[-78.0, 23.0]'
2019-08-11T17:06:21.0730826Z E       AssertionError: assert '[23.0, -78.0]' == '[-78.0, 23.0]'
2019-08-11T17:06:21.0730921Z E         - [23.0, -78.0]
2019-08-11T17:06:21.0730994Z E         + [-78.0, 23.0]
```

**Comment by snowman2** [comment]
Hmmm, I think in the tests on the rasterio repo they are using PROJ 6.1.1. Not sure that matters, just noting the difference.

**Comment by ocefpaf** [comment]
> Hmmm, I think in the tests on the rasterio repo they are using PROJ 6.1.1. Not sure that matters, just noting the difference.

I noticed that and sent https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/264 to test it. I'm trying it locally first, before starting migrator.

**Comment by ocefpaf** [comment]
Upstream is not testing this on Python 3.7. See https://github.com/mapbox/rasterio/blob/096cb4878fd3490bad52e17835285de8a2aeeebf/.travis.yml#L5-L7. There aren't tests for Windows too :-/

@snowman2 I'm not sure if we should merge this to have some `rasterio` with `gdal 3` or if we should wait for a new release.

**Comment by snowman2** [comment]
Definitely a toss up. What are the downsides to merging now?

**Comment by ocefpaf** [comment]
> Definitely a toss up. What are the downsides to merging now?

We won't ship Python 3.7 on Linux and the Windows packages but the rest should be OK. We are skipping `test_transform_geom_polygon_cutting` and `test_transform_geom_polygon_offset` on OS X, let me try to run those again to remember why and/or if that may affect Python 3.7 there too.

**Comment by snowman2** [comment]
@conda-forge-admin please rerender.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -58,7 +58,7 @@ test:
     - rio --help
     - rio info {{ RECIPE_DIR }}/test_data/test.tif  # [not win]
     - rio info {{ RECIPE_DIR }}\test_data\test.tif  # [win]
-    - pytest -s -rxs -v -m "not wheel" tests  # [linux]
+    - pytest -s -rxs -v -m "not wheel" tests  # [not win]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Argh. Thanks!

Comment (snowman2) [reply recipe/meta.yaml]: Not a problem 😄

**Comment by ocefpaf** [comment]
It is easier to re-submit this one.

**Pull request closed without merging**